plugins/inmemoryscanner/test/Scanner_unittest.cpp

```diff
@@ -245,9 +260,13 @@ namespace InMemoryScanner
     {
         auto processWithShortName = getProcessInfoFromRunningProcesses(testPid);
         auto expectedProcessName = processWithShortName->name;
-        auto expectedFileNameRegEx = expectedProcessName + "-" + std::to_string(testPid) + "-" + protectionAsString +
-                                     "-" + intToHex(startAddress) + "-" + intToHex(startAddress + size) + "-" +
-                                     uidRegEx;
+        auto expectedFileNameRegEx = fmt::format("{}-{}-{}-{:x}-{:x}-{}",
```

Comment:
nit: why not use the getMemFileName function?

Reply:
The what?